revolut/__init__.py

```diff
@@ -166,6 +166,66 @@ def transaction(self, id):
     # TODO: def transaction_by_request_id(self, request_id):
 
 
+class ClientMerchant(utils._SetEnv):
+    live = False
+    _session = None
+
+    def __init__(self, session):
+        self._set_env(session.access_token)
+        self._session = session
+
+    def _request(self, func, path, data=None):
```

Comment:
Myślę że sporo kodu z obsługi requestów będzie można wydzielić do jakiegoś `class BaseClient`. Ale to później.